### PR TITLE
Updates release state and current branches

### PR DESCRIPTION
**Pull request by benskelker**
DO NOT MERGE THIS PR UNTIL RELEASE DAY

### CODE REVIEW

**Comment by benskelker** [comment]
@webmat 
Should I update the ECS version in the 7.6.asciidoc file to 1.4?

```:ecs_version:            1.3```

**Comment by benskelker** [comment]
Failing - no [azure-marketplace](https://github.com/elastic/azure-marketplace) 7.6 branch

**Comment by webmat** [comment]
@benskelker Yes please, update it to 1.4!

We initially thought Beats wouldn't go to 1.4 for 7.6, but they imported it in time for the feature freeze :-)

File: conf.yaml [hunk 1]
```diff
@@ -180,7 +180,7 @@ contents:
           -
             title:      Azure Marketplace and Resource Manager (ARM) template
             prefix:     en/elastic-stack-deploy
-            current:    7.5
+            current:    7.6
```
Comment (lcawl) [inline conf.yaml]: This branch of documentation doesn't exist yet and so it's causing this build err:
> 06:49:06 INFO:build_docs:Current branch <7.6> is not in <branches> in book <Azure Marketplace and Resource Manager (ARM) template> at lib/ES/Book.pm line 114.

Comment (benskelker) [reply conf.yaml]: Thanks @lcawl 
Is there going to be an Azure Marketplace and Resource Manager (ARM) template branch? Do you know who I can ask? If there is, maybe I'll just leave it and retest later.

Comment (benskelker) [reply conf.yaml]: Confirmed by @russcam - this trails the stack release. Set current to 7.5.

**Comment by benskelker** [comment]
Failed due to broken links in ES ref guide. I'll retry when those are fixed.

**Comment by benskelker** [comment]
run elasticsearch-ci/docs

**Comment by benskelker** [comment]
@lcawl 
Hi - can you take a look at the last commit. Not sure it's right.

Thanks

**Comment by bmorelli25** [comment]
I'm doing a bit of a retrospective here. First off, the bad APM links are definitely my fault. The separately versioned Agents are commonly an issue during release time, and it would've been prudent of me to run a full build ahead of time to make sure we were good.

As a secondary line of defense, and @lcawl / @nik9000 correct me if I'm wrong, I think we also could've seen the bad links in this PR. If I recall, the docs repo only runs a regular doc build. But we have the option of also running `@elasticmachine, run elasticsearch-docs/doc rebuild` (emphasis on the **rebuild**). That would've attempted a rebuild of this PR and shown us the broken links ahead of time? Does that sound correct? If so, I'll update the wiki to include this step in future releases.

**Comment by lcawl** [comment]
Yes our wiki says:

> To force all versions to be rebuilt, you can do:
> @elasticmachine run elasticsearch-ci/docs rebuild

I agree that we should consider adding that to our release processes.  I also think we should document those commands in our docs readme, so they're not buried and forgotten in internal wikis

**Pull request merged**